### ISSUE TITLE
[java] CommentDefaultAccessModifier false positive with nested interfaces (regression from 6.4.0)

### ISSUE THREAD

**Issue by boris-petrov** [pmd/pmd#1211]
On PMD 6.5.0, having an interface in an interface causes that warning:

```java
public interface A {
	interface B {
		default void b() {}
	}
}
```

This was not so in 6.4.0.

**Comment by adangel** [comment]
Do you mean **6**.5.0? If yes -> awesome for testing the new version already! If not, could you please retest with 6.5.0? Thanks!

**Comment by boris-petrov** [comment]
I've edited the comment. Absolutely it is 6.5.0, sorry for the confusion. :)

**Comment by adangel** [comment]
Thanks! Yes, this seems indeed to be an oversight.
